lib/config.mk.in

```diff
@@ -5,6 +5,6 @@ CUDA_OBJS := @CUDA_OBJS@
 DEFINE_MACROS := @DEFINE_MACROS@
 prefix := @prefix@
 exec_prefix := @exec_prefix@
-CFLAGS := @MKCFLAGS@ -I@includedir@
+CFLAGS := @CPPFLAGS@ @MKCFLAGS@ -I@includedir@
```

Comment:
Oh I just want this to work:
```bash
CPPFLAGS="-I/opt/local/include" LDFLAGS="-L/opt/local/lib" ./configure
```
When configure is searching for headers it uses the CPPFLAGS environment variable,
so the configure works, but if you don't add CPPFLAGS to the CFLAGS make variable
in config.mk it will fail to compile even though configure works.

See: http://www.gnu.org/software/autoconf/manual/autoconf-2.66/html_node/Preset-Output-Variables.html